src/middlewared/middlewared/alembic/versions/25.04/2024-12-16_12-49_enclosure_label.py

```diff
@@ -0,0 +1,28 @@
+"""Rename `enclosure_label` table
+
+Revision ID: 19cdc9f2d2df
+Revises: b44c092bfa30
+Create Date: 2024-12-16 12:49:19.950812+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '19cdc9f2d2df'
+down_revision = 'b44c092bfa30'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("ALTER TABLE truenas_enclosurelabel RENAME TO enclosure_label")
```

Comment:
The table will have entries with `encid`'s that don't match with what `enclosure2.query` returns so how does a user remove those entires?

Reply:
We can automatically remove the labels for non-existing IDs on middleware startup. Will this be safe?